### PR TITLE
strings: tr: Update locales

### PR DESCRIPTION
**Pull request by windowz414**
* Updated old strings from old locales repo.

* Added new strings that were introduced as per commit a77c7dea541e6f31eb3ad903498eae061418a08b (Support: L10n support and rewriting responses).

Signed-off-by: Bedirhan KURT <windowz414@gnuweeb.org>

### CODE REVIEW

File: locales/tr.strings [hunk 1]
```diff
@@ -102,7 +102,6 @@
 "write" = "Yaz";
 "publish" = "Yayınla";
 "delete" = "Sil";
-"comments" = "Yorumlar";
```
Comment (veselcraft) [inline locales/tr.strings]: Why?

**Review by veselcraft — CHANGES_REQUESTED** [review]
No need to delete the Comments string

File: locales/tr.strings (hunk 1 above)
Comment (windowz414) [reply locales/tr.strings]: > No need to delete the Comments string

Then why was it deleted here?

https://github.com/windowz414/openvk/commit/a77c7dea541e6f31eb3ad903498eae061418a08b#diff-12b70ca9f0009f215fc01a7bc214dde1bbe09335e4ee010a6ad2a8ada48fc953L116

**Comment by veselcraft** [comment]
nvm

**Review by veselcraft — APPROVED** [review]

**Pull request merged**